conf/up4.bess

```diff
@@ -204,6 +204,42 @@ _in -> qerLookup::Qos(fields=[ {'attr_name':'src_iface', 'num_bytes':1}, \
                                {'attr_name':'fseid', 'num_bytes':8}], \
                              values=[{'attr_name':'qfi', 'num_bytes':1}])
 
+executeFAR::Split(size=1, attribute='action')
+afterFarMerge = executeFAR
```

Comment:
Maybe move the rate limiter after the preQosCounter is better:
- avoids extra work for dropped packets
- no need to account for gtp header overhead

Reply:
I think the placement is a off compared to the intent above. What you describe above is what we want. All metering to happen together and on un-encapsulated traffic.

![image](https://user-images.githubusercontent.com/6157640/131949806-e500fe29-a4da-4b49-8e8b-0dfd976331f8.png)

Reply:
I can see now why the programming of entries will need to go into the agent. The GBR flows cannot be dropped after QERLookup. So basically the rest of bandwidth available on the slice need to be the entry. Since GBR flows are added dynamically, the remaining bandwidth needs to updated in the entry accordingly.

cc @badhrinathpa

Reply:
We need to work our way backwards from total available wire bandwidth (extra 20B) per slice, for each logical interface N3/N6/N9 because even though N6 and N9, both say core, N6 does not have GTPu overhead and N9 does.